package.json

```diff
@@ -1136,45 +1193,59 @@
 				"description": "%copilot.toolSet.editing.description%",
 				"icon": "$(pencil)",
 				"tools": [
-					"createFile",
 					"createDirectory",
+					"createFile",
```

Comment:
these are sorted alphabetically, not sure if there was intent behind the original order?

Reply:
I don't think so